test/validators/htmlValidator.js

```diff
@@ -39,10 +39,22 @@ describe('HTMLValidator', function () {
         const routeInfo = new RouteInfo(path, method);
         const htmlInfoDictionary = parseHTML(this.yamlData.paths[path][method].responses, this.resources.views, this.resources.layouts);
         const output = await this.htmlValidator.validate({routeInfo, htmlInfoDictionary});
+
         output.should.be.a.string;
         output.should.include('missing-template.pug');
         output.should.include('missing-html.html');
       });
+
+      it('Should return an error string when there is an unavailable layout', async function () {
```

Comment:
Why is this tested and not the  example given in the issue?

Reply:
Could you link this example?